MC_Mix.cmd

```diff
@@ -173,37 +173,34 @@ if contains("$scriptlist", "mastercraft") then
 	{
 	action (work) off
 	var temp.room $roomid
-     if ((%water.gone = 1) || (%alcohol.gone = 1) || (%catalyst.gone = 1)) then
-          {
-               if ("$righthandnoun" = "%bowl") then gosub STOW_RIGHT)
-               if ("$lefthandnoun" = "%bowl") then gosub STOW_LEFT)
-          }
+	if ((%water.gone = 1) || (%alcohol.gone = 1) || (%catalyst.gone = 1)) then
+        {
+                if ("$righthandnoun" = "%bowl") then gosub STOW_RIGHT
+                if ("$lefthandnoun" = "%bowl") then gosub STOW_LEFT
+        }

```

Comment:
In das/master there is a typo. I had more complex logic in here, but it was manually replaced with the more elegant stow routines. Unfortunately there is a bug (extra bracket)